bin/ofs/Cargo.toml

```diff
@@ -46,8 +46,8 @@ tokio = { version = "1.37.0", features = [
 ] }
 url = "2.5.0"
 
-[target.'cfg(any(target_os = "linux", target_os = "freebsd"))'.dependencies]
-fuse3 = { "version" = "0.7.2", "features" = ["tokio-runtime", "unprivileged"] }
+[target.'cfg(any(target_os = "linux", target_os = "freebsd", target_os = "macos"))'.dependencies]
+fuse3 = { git = "https://github.com/oowl/fuse3", branch = "owl/macos-support-recreate", "features" = ["tokio-runtime", "unprivileged"] }
```

Comment:
Hi, we will need to wait for new release of fuse3.

Reply:
Yeah, we need to wait https://github.com/Sherlock-Holo/fuse3/pull/108 merged to fuse3 upstream.